src/components/Trip/TripView.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useMemo } from 'react'
+import { useLocation } from 'react-router-dom'
+
+import { isQueryLoading, useQuery, hasQueryBeenLoaded } from 'cozy-client'
+import Spinner from 'cozy-ui/transpiled/react/Spinner'
+
+import TripLayout from 'src/components/Trip/TripLayout'
+import { buildGeojsonQuery } from 'src/queries/queries'
+import { transformTimeSeriesToTrips } from 'src/lib/trips'
+
+const TripView = () => {
+  const { pathname } = useLocation()
+  const geojsonId = useMemo(() => pathname.split('/').pop(), [pathname])
+
+  const geoJsonQuery = buildGeojsonQuery(geojsonId)
+  const { data, ...tripQueryResult } = useQuery(
+    geoJsonQuery.definition,
+    geoJsonQuery.options
+  )
+
+  const trip = useMemo(() => {
+    if (!data || !data.length) {
+      return null
+    } else {
+      return transformTimeSeriesToTrips(data)[0]
+    }
+  }, [data])
+
+  const isLoading =
+    isQueryLoading(tripQueryResult) && !hasQueryBeenLoaded(tripQueryResult)
```

Comment:
It means that if the query has been at least loaded once, then when coming back after the `fetchPolicy` ended, we will not display the Spinner but the data we already loaded and in the same time launching the request, and then we'll update the data when the request is resolved. 

We can have a "flickering" for the user. Is it really what we want?